app/src/main/java/org/jellyfin/androidtv/util/profile/ProfileHelper.kt

```diff
@@ -15,6 +15,34 @@ import timber.log.Timber
 object ProfileHelper {
 	private val MediaTest by lazy { MediaCodecCapabilitiesTest() }
 
+	private val supports4K by lazy {
+		val (maxWidth, maxHeight) = MediaTest.getMaxResolution()
+		maxWidth >= 3840 && maxHeight >= 2160
+	}
+
+	val max1080pCodecProfile by lazy {
```

Comment:
Quoting from my previous review:

> It would be even better to use the highest supported width/height values directly in the profile instead.

Reply:
I integrated it into the `max1080pCodecProfile`.

Reply:
No you did not? It still hardcodes the width and height based on the result of the getMaxResolution function.

Reply:
Should be removed now.